### PR TITLE
added pong command

### PR DESCRIPTION
**Pull request by matrom01-v2**

### CODE REVIEW

File: src/commands/pong.ts
Comment (raspberry-varg) [inline src/commands/pong.ts:1]: Run Prettier on your file.
Install the Prettier extension on Visual Studio Code and run it on a file with Shift+Alt+F.

**Review by raspberry-varg — CHANGES_REQUESTED** [review]
Changes pending; Prettier needs to be ran.

File: src/commands/pong.ts
Comment (matrom01-v2) [reply src/commands/pong.ts:1]: fixed with prettier

**Review by raspberry-varg — APPROVED** [review]
Looks great!

**Pull request merged**